wafer/users/models.py

```diff
@@ -22,6 +23,7 @@
 # Specification taken from https://support.twitter.com/articles/101299
 TwitterValidator = RegexValidator('^[A-Za-z0-9_]{1,15}$',
                                   'Incorrectly formatted twitter handle')
+is_registered = import_string(settings.WAFER_USER_IS_REGISTERED)
```

Comment:
Is it worth catching ImportError and returning a message that explicitly points to the WAFER_USER_IS_REGISTERED setting, rather than just the raw error?